application/src/main/java/org/togetherjava/tjbot/features/github/GitHubCommand.java

```diff
@@ -111,7 +123,7 @@ public void onAutoComplete(CommandAutoCompleteInteractionEvent event) {
             event.replyChoiceStrings(choices).queue();
         }
 
-        if (lastCacheUpdate.isAfter(Instant.now().minus(CACHE_EXPIRES_AFTER))) {
+        if (lastCacheUpdate.isBefore(Instant.now().minus(CACHE_EXPIRES_AFTER))) {
```

Comment:
not your fault, but this condition messes hard with my brain. heck, im not even sure that your change is correct. lets just rewrite it to make it easier to understand and then verifying its correctness is much easier as well. maybe:

```java
Instant cacheExpiredAt = lastCacheUpdate.plus(CACHE_EXPIRES_AFTER);
boolean isCacheExpired = Instant.now().isAfter(cacheExpiredAt);
if (isCacheExpired) {
  ...
}
```

You can also put it into a quick helper method:

```java
private boolean isCacheExpired() {
  Instant cacheExpiredAt = lastCacheUpdate.plus(CACHE_EXPIRES_AFTER);
  return Instant.now().isAfter(cacheExpiredAt);
}
```
and then use if `if (isCacheExpired()) {...}`